### PR TITLE
backupccl: pause online restore if disk is full

### PR DESCRIPTION
**Pull request by dt**
Release note: none.
Epic: none.

### CODE REVIEW

**Review by msbutler — COMMENTED** [review]
not sure why this doesn't work :D

File: pkg/server/span_download.go [hunk 1]
```diff
@@ -87,16 +89,44 @@ func (s *systemStatusServer) localDownloadSpan(
...
+				defer close(stopDiskMonCh)
+				return downloadSpans(ctx, e, downloadSpansCh, req.ViaBackingFileDownload)
 			},
 			// Send spans to downloadSpansCh.
 			func(ctx context.Context) error {
 				defer close(downloadSpansCh)
 				return sendDownloadSpans(ctx, req.Spans, downloadSpansCh)
 			},
+			func(ctx context.Context) error {
+				tick := time.NewTicker(time.Second * 15)
```
[the remark this answers is not in the document]
Comment (msbutler) [reply pkg/server/span_download.go:106]: yeah fine, leave it as is. more trouble than it's worth

**Comment by dt** [comment]
> not sure why this doesn't work

Turns out Download communicates all its errors back to its caller in plain untyped strings. Fix incoming.

**Review by msbutler — APPROVED** [review]

**Comment by dt** [comment]
TFTR!

bors r+

**Pull request merged**